modules/localstack/examples_test.go

```diff
@@ -113,3 +116,83 @@ func ExampleRunContainer_legacyMode() {
 	// Output:
 	// version=localstack/localstack:0.10.0. Testcontainers for Go does not support running LocalStack in legacy mode. Please use a version >= 0.11.0
 }
+
+func ExampleRunContainer_UsingLambdas() {
+	ctx := context.Background()
+
+	flagsFn := func() string {
+		labels := testcontainers.GenericLabels()
+
+		flags := ""
+		for k, v := range labels {
+			flags = fmt.Sprintf("%s -l %s=%s", flags, k, v)
+		}
+
+		return flags
+	}
+
+	lambdaName := "localstack-lambda-url-example"
+
+	container, err := localstack.RunContainer(ctx,
+		testcontainers.WithImage("localstack/localstack:2.3.0"),
+		testcontainers.CustomizeRequest(testcontainers.GenericContainerRequest{
+			ContainerRequest: testcontainers.ContainerRequest{
+				Env: map[string]string{
+					"SERVICES":                 "lambda",
+					"LAMBDA_EXECUTOR":          "docker",
+					"PROVIDER_OVERRIDE_LAMBDA": "asf",
```

Comment:
I think those are not required in LS 2.3.0

Reply:
Cool! I tried locally without LAMBDA_EXECUTOR and PROVIDER_OVERRIDE_LAMBDA and it works! Thanks!